Documentation/ApiOverview/Fal/UsingFal/Tca.rst

```diff
@@ -14,79 +14,32 @@ create relations to files.
     :php:`ExtensionManagementUtility::getFileFieldTCAConfig()` was used to
     create a TCA configuration suitable to handle files. This function has been
     deprecated with the introduction of the new field type :ref:`t3tca:columns-file`.
+    See the :ref:`Migration <t3tca:columns-file-migration>` section on how to
+    adjust the configuration.
 
 The TCA field type :ref:`t3tca:columns-file` can be used to provide a field
-in which files can be referenced and or uploaded:
+in which files can be referenced and/or uploaded:
 
-..  code-block:: php
-    :caption: EXT:some_extension/Configuration/TCA/my_table.php
+..  literalinclude:: _Tca/_my_table.php
+    :language: php
+    :caption: EXT:my_extension/Configuration/TCA/my_table.php
 
-    return [
-        'columns' => [
-            'my_media_file' => [
-                'label' => 'My image',
-                'config' => [
-                    'type' => 'file',
-                    'allowed' => 'common-media-types'
-                ],
-            ],
-        ],
-        // ...
-    ];
-
-On the database side, the corresponding field needs just store an integer,
+On the database side, the corresponding field needs to store an integer,
 as is usual for relations field:
 
-.. code-block:: sql
-   :caption: EXT:some_extension/ext_tables.sql
-
-   CREATE TABLE my_table (
-      my_media_file int(11) unsigned DEFAULT '0' NOT NULL,
-   );
+..  literalinclude:: _Tca/_ext_tables.sql
+    :language: sql
+    :caption: EXT:my_extension/ext_tables.sql
 
-The property :php:`appearance` can be used to specify if a file upload button
-and file by URL button (Vimeo, Youtube) should be displayed.
+The property :ref:`appearance <t3tca:columns-file-properties-appearance>` can be
+used to specify, if a file upload button and file by URL button (Vimeo, Youtube)
+should be displayed.
 
 Example:
 
-..  code-block:: php
-    :caption: EXT:some_extension/Configuration/TCA/Overrides/my_table.php
-
-    $GLOBALS['TCA']['my_table']['columns']['my_media_file']['config']['appearance'] = [
-        'fileUploadAllowed' => false,
-        'fileByUrlAllowed' => false,
-    ];
+..  literalinclude:: _Tca/_overrides_my_table.php
+    :language: php
+    :caption: EXT:my_extension/Configuration/TCA/Overrides/my_table.php
 
 This will suppress two buttons for upload and external URL and only leave
 the button :guilabel:`Create new relation`.
-
-Migration from :php:`ExtensionManagementUtility::getFileFieldTCAConfig`
```

Comment:
I would like to leave this in for TYPO3 v11 and v12. I would agree about removing it for 13

Reply:
I linked it in the versionchanged directive above, so no need to have this twice in the docs.

Reply:
I would rather have it in TYPO3 explained as well as this will probably confuse devs and a small link can easely be overlooked

Reply:
Done.